server/routes/logins.js

```diff
@@ -48,14 +76,13 @@ router.post('/register', (req, res) => {
 
 router.post('/login', (req, res, next) => {
   console.log("test11111🤒");
+  console.log(req.user)
   passport.authenticate('local', (err, user, info) => {
     if(err) {
-      console.log('1')
       return next(err)
     }
     if(!user) {
-      console.log('2')
-      res.status(401).send('Incorrect Password');
+      res.status(401).send(info.message);
     }else {
       // const {first_name, last_name, email, id, balance} = user;
```

Comment:
seems there are a lot of console.logs you could remove once you know the features working